ibtk/include/ibtk/FECache.h

```diff
@@ -54,6 +54,22 @@ namespace IBTK
 
 /////////////////////////////// PUBLIC ///////////////////////////////////////
 
+/**
+ * Enumeration describing the various update options available for
+ * libMesh::FEBase objects stored by FECache. Multiple flags can be enabled
+ * with bitwise or operations, e.g.,
+ *
+ * @code
+ * const IBTK::FEUpdateFlags update_flags = IBTK::update_phi | IBTK::update_dphi;
+ * @endcode
+ */
+enum FEUpdateFlags
+{
+    update_default = 0,
```

Comment:
What is computed for `default`?

Reply:
Nothing. I should go back and label these.